cookbooks/hangops-jobbot/recipes/default.rb

```diff
@@ -14,6 +14,12 @@
 # limitations under the License.
 #
 
+# Create a Hubot user
+user 'hubot' do
+   action :create #default action
+   group "groupid"
```

Comment:
You have any user groups on the host? group is not necessary if you don't want to add it to a user group